presto-product-tests/src/main/java/com/facebook/presto/tests/ImpersonationTests.java

```diff
@@ -15,29 +15,33 @@
 
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
+import com.teradata.tempto.BeforeTestWithContext;
 import com.teradata.tempto.ProductTest;
 import com.teradata.tempto.Requires;
 import com.teradata.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableNationTable;
 import com.teradata.tempto.hadoop.hdfs.HdfsClient;
+import com.teradata.tempto.query.QueryExecutor;
 import org.testng.annotations.Test;
 
 import static com.facebook.presto.tests.TestGroups.HDFS_IMPERSONATION;
 import static com.facebook.presto.tests.TestGroups.HDFS_NO_IMPERSONATION;
 import static com.facebook.presto.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
-import static com.teradata.tempto.query.QueryExecutor.query;
+import static com.facebook.presto.tests.utils.QueryExecutors.connectToPresto;
 import static java.lang.String.format;
 import static org.testng.Assert.assertEquals;
 
 @Requires(ImmutableNationTable.class)
 public class ImpersonationTests
         extends ProductTest
 {
+    private QueryExecutor aliceExecutor;
+
     @Inject
     private HdfsClient hdfsClient;
 
     @Inject
-    @Named("databases.presto.jdbc_user")
-    private String prestoJdbcUser;
+    @Named("databases.alice@presto.jdbc_user")
```

Comment:
Could you please extract the changes to this file to a separate commit.

Reply:
done.